extensions/ql-vscode/gulpfile.ts/tests.ts

```diff
@@ -6,6 +6,10 @@ export function copyTestData() {
   return Promise.resolve();
 }
 
+export function watchTestData() {
+  return gulp.watch(['src/vscode-tests/no-workspace/data/**/*', 'src/vscode-tests/minimal-workspace/data/**/*'], copyTestData);
```

Comment:
Maybe we should add a reminder in the contributing docs that all test data (sample projects, config files, etc) must go in the `data` folder.

And maybe this is a slight simplification that will make it easier to read:

```suggestion
  return gulp.watch('src/vscode-tests/*/data/**/*', copyTestData);
```